package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "debugger",
-  "displayName": "temporal-debugger-extension",
-  "description": "Temporal VSCode Debugger extension",
+  "name": "temporal-vscode-extension",
```

Comment:
```suggestion
  "name": "temporalio",
```

i like the brevity ("go install `temporalio.temporal-technologies`" versus "go install `temporal-vscode-extension.temporal-technologies`"). feel free to reject if it's a pain to unpublish the existing one.

sadly just `temporal` is taken, both name and displayName. i asked the marketplace to release the name to us, but haven't heard back yet

<img width="382" alt="image" src="https://user-images.githubusercontent.com/251288/235711913-229fa776-99a2-4aaf-8bd7-6c339cfd34f5.png">

Reply:
That's fair! I think I can just remove and republish. Updated ✅